check_manifest.py

```diff
@@ -340,6 +355,9 @@ def _list_submodules(cls):
         # there's no .gitmodules file.
         if not os.path.exists('.gitmodules'):
             return []
+        # Bug: git submodule will print /paths/like/this when you run it from
+        # an msys shell (like Git Bash), and we will fail then.  See
+        # https://github.com/mgedmin/check-manifest/issues/64
         return run(['git', 'submodule', '--quiet', 'foreach', '--recursive',
                     'printf "%s/%s\\n" $toplevel $path'], encoding=cls._encoding).splitlines()
```

Comment:
An alternative fix may be possible where I strip the $toplevel of the toplevel repo myself instead of relying on `os.path.relpath()` to do that for me, assuming I can figure what the $toplevel is.  Something like

```python
toplevel = run(['git', 'submodule', '--quiet', 'printf "$toplevel/\n"', encoding=...].splitlines()
if not toplevel:
    return []
toplevel = toplevel[0]
submodules = run(['git', 'submodule', '--quiet', 'foreach', '--recursive', 'printf "%s/%s\\n" $toplevel $path'], encoding=cls._encoding).splitlines()
return remove_prefix_from_each_line(toplevel, submodules)
```